src/components/modules/epics/ContributionsEpicACAbove.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import { EpicSeparateArticleCountTestVariants } from '../../../tests/epicArticleCountTest';
+import { ContributionsEpicComponent, EpicProps } from './ContributionsEpic';
+
+export const ContributionsEpic: React.FC<EpicProps> = ContributionsEpicComponent(
+    EpicSeparateArticleCountTestVariants.above,
+);
```

Comment:
I think the naming of `ContributionsEpicACAbove` vs `ContributionsEpicArticleCountAbove` is a tad unclear. However, I guess this code will all change after the ab test so maybe it's not really worth spending too much time on naming just yet!

Reply:
you're right, it's very confusing. I've just renamed to e.g. `ContributionsEpicWithArticleCountAbove`

Reply:
Oh yeah, that's a great shout 👍